hrishikeshpaul/vibey
# Create room and tags, with backend integration to retrieve tags

Target Issues: #20 

### How to test it?
- Login
- Click on the "create room"
- Enter the details
- Make sure you have a few tags in the database or use `http://localhost:5555/api/tag/add?name=<name>` to add a few tags


### Known issue
For some reason I can't get the focus border to appear on the tag input field. Click on the name or description and you'll see an outline i vibey's primary color. Doesn't happen for input, might need help to fix that
